### PR TITLE
Add drag'n'drop attribute reordering

### PR DESCRIPTION
**Pull request by dominik-zeglen**
I want to merge this change because it allows users to reorder attributes in product type view.

### Pull Request Checklist

1. [ ] Privileged views and APIs are guarded by proper permission checks.
1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Database queries are optimized and the number of queries is constant.
1. [ ] Database migration files are up to date.
1. [ ] The changes are tested.
1. [ ] The code is documented (docstrings, project documentation).
1. [ ] GraphQL schema and type definitions are up to date.
1. [x] Changes are mentioned in the changelog.

### CODE REVIEW

**Comment by django-queries** [comment]
Here is the report for a80fcd8f6f3fbc0bc64e22318ff8ab44a7dedff2 (mirumee/saleor @ attributes/add-dnd)
Base comparison is 09f65d64875a1e9d8450b6cd255216ffd8c7ee9e.



<details><summary>No differences were found. (click me)</summary>
<p>

```diff
# api.benchmark checkout
  test name                           	left count 	right count	duplicate count
  ------------------------------------	-----------	-----------	---------------
  add billing address to checkout     	         34	         34	             20
  add shipping to checkout            	          7	          7	              0
  checkout payment charge             	         14	         14	              0
  complete checkout                   	          6	          6	              0
  create checkout                     	         43	         43	             21

# api.benchmark homepage
  test name                           	left count 	right count	duplicate count
  ------------------------------------	-----------	-----------	---------------
  retrieve main menu                  	          5	          5	              0
  retrieve product list               	          4	          4	              0
  retrieve secondary menu             	          5	          5	              0
  retrieve shop                       	          2	          2	              0

# api.benchmark product
  test name                           	left count 	right count	duplicate count
  ------------------------------------	-----------	-----------	---------------
  product details                     	         15	         15	              3

# api.benchmark variant
  test name                           	left count 	right count	duplicate count
  ------------------------------------	-----------	-----------	---------------
  retrieve variant list               	         15	         15	              6
```

</p>
</details>

File: saleor/static/dashboard-next/components/SortableTable/SortableHandle.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "&:first-child": {
+      paddingRight: theme.spacing.unit * 2
+    },
+    cursor: "grab",
+    width: 48 + theme.spacing.unit * 1.5
+  }
+}));
+
+const SortableHandle = SortableHandleHoc(() => {
+  const classes = useStyles({});
```
Comment (benekex2) [inline saleor/static/dashboard-next/components/SortableTable/SortableHandle.tsx]: empty styles

File: saleor/static/dashboard-next/components/SortableTable/SortableTableBody.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    fontFamily: theme.typography.fontFamily,
+    fontSize: theme.overrides.MuiTableCell.root.fontSize,
+    opacity: 0.5
+  }
+}));
+
+const SortableTableBody: React.FC<
+  TableBodyProps & SortableTableBodyProps
+> = props => {
+  const classes = useStyles({});
```
Comment (benekex2) [inline saleor/static/dashboard-next/components/SortableTable/SortableTableBody.tsx]: empty styles

File: saleor/static/dashboard-next/components/SortableTable/SortableHandle.tsx (hunk 1 above)
Comment (dominik-zeglen) [reply saleor/static/dashboard-next/components/SortableTable/SortableHandle.tsx]: Empty props, `@material-ui/styles/makeStyles` is poorly typed and requires argument here even if not used

**Comment by maarcingebala** [comment]
I'm merging this with CI failing on Python tests. @NyanKiyoshi will look into it.

**Pull request merged**